src/website/app/demos/Dialog/examples/DialogTitle/variants.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import { DialogHeader, DialogTitle } from '../../../../../../library/Dialog';
+import { capitalize } from '../../../../../../library/utils';
+import DemoLayout from '../../components/DemoLayout';
+import Dialog from '../../components/DemoDialog';
+
+export default {
+  id: 'variants',
+  title: 'Variants',
+  description: `Use the \`variant\` prop to help
+[communicate intent](/color#guidelines-variants).
+
+  <Callout title="Note">
+    <p key={0}>
+      Note that <code key={0}>variant</code> is automatically supplied to the
+      title if the prop is passed to the Dialog root;
+      see <a href="/components/dialog#variants" key={1}>Dialog Variants</a>.
+    </p>
```

Comment:
I think I added an unnecessary "Note that" since this is already a Note Callout. Also, should we add a similar callout in the DialogActions variants example?